contracts/encoders/evm-encoder/src/libraries/aave_position_manager.rs

```diff
@@ -0,0 +1,119 @@
+use alloy_sol_types::{SolCall, SolValue};
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Binary, StdError, StdResult, Uint256};
+
+use valence_encoder_utils::libraries::{
+    aave_position_manager::solidity_types::{
+        borrowCall, repayCall, repayWithATokensCall, supplyCall, withdrawCall,
+    },
+    updateConfigCall,
+};
+use valence_library_utils::{msg::ExecuteMsg, LibraryAccountType};
+
+use crate::parse_address;
+
+use super::{get_update_ownership_call, get_update_processor_call};
+
+// We need to define a config and functions for this library as we don't have a CosmWasm equivalent
+#[cw_serde]
+/// Struct representing the library configuration.
+pub struct LibraryConfig {
+    /// The input address for the library.
+    pub input_addr: LibraryAccountType,
+    /// The output address for the library.
+    pub output_addr: LibraryAccountType,
+    /// The AAVE pool address.
+    pub aave_pool: String,
```

Comment:
feels like it's clear that it's an aave pool given that its coming from `aave_position_manager.rs`
```suggestion
    pub pool: String,
```